index.html

```diff
@@ -4,38 +4,42 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="style.css" />
     <title>mobile version</title>
   </head>
 
   <body>
-    <header class="top-head">
-      <div class="navbar">
-        <a class="navbar-container">Arrow Dan</a>
-        <a href="#">
-          <img
-            class="hamburger hide-desktop"
-            src="image/Union.png"
-            alt="navigation icon"
-          />
-        </a>
-      </div>
+    <header class="header">
+      <a class="container" href="index.html">Arrow Dan</a>
+      <nav class="navbar">
+        <a href="#"><img
+          class="hamburger hide-desktop"
+          src="image/Union.png"
+          alt="navigation icon"
+        /></a>
+        <ul class="hide-mobile">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
```

Comment:
[REQUIRED] The headline text is not perfectly centered like in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278). I should see all the text when I'm on the homepage on 100% page view.
![image](https://user-images.githubusercontent.com/38088818/165911261-6875f67f-531c-4e0b-93e1-fa0e862b51fc.png)

Figma Template:
![image](https://user-images.githubusercontent.com/38088818/165911369-1566baee-ea97-4f5a-8a53-280a12fbaeb6.png)

<br />
<br />
<br />

**[OPTIONAL] The following are only optional because they were not mentioned by the previous reviewer. However, it is part of the project requirements and I strongly recommend that you take them into consideration.**

1. [OPTIONAL] You are missing the mail icon on the navbar, and your items are at the bottom of the navbar:
![image](https://user-images.githubusercontent.com/38088818/165914960-8aa06cfc-b176-47c5-9868-01fee50e6940.png)

Figma Navbar:
![image](https://user-images.githubusercontent.com/38088818/165915478-8281f834-245f-4479-a308-9bc3189011be.png)

<br />
<br />
<br />
2. [OPTIONAL] The `My recent works` section is skewed to the right leaving a gap on the left. The must be margins on both sides making the works section perfectly centered on 100% page view.
- Also the programming languages boxes overlap with the project description text.

![image](https://user-images.githubusercontent.com/38088818/165912261-307f3ab2-fda9-4d4d-8c32-5741f1487ac1.png)

Figma Template:
![image](https://user-images.githubusercontent.com/38088818/165912500-5c9e3b5f-a69b-480f-9c57-c09fd839e6f0.png)

<br />
<br />
<br />
3. [OPTIONAL] The lists of languages, frameworks, and skills section are also skewed to the right, leaving a gap on the left. This section must be perfectly centered on 100% page view.

![image](https://user-images.githubusercontent.com/38088818/165912970-fa9665fb-1015-43ad-9a2e-ae7b50d0cdab.png)

Figma Template:
![image](https://user-images.githubusercontent.com/38088818/165913541-4cd6aaf5-1641-4c9f-b51c-eb01344451d0.png)

<br />
<br />
<br />
4. [OPTIONAL] The same thing applies to your contact form:

![image](https://user-images.githubusercontent.com/38088818/165913912-1e8e8d88-a07d-442b-b08a-a1ae911941c9.png)

Figma Template:
![image](https://user-images.githubusercontent.com/38088818/165914116-2e5f73c4-47a9-4233-af53-935dbd1ee189.png)